custom_components/jcihitachi_tw/__init__.py

```diff
@@ -243,4 +247,4 @@ def update(self):
         )
         
         # Important: We have to reset the update scheduler to prevent old status from wrongly being loaded. 
-        self.coordinator.async_set_updated_data(None)
+        self.coordinator.set_updated_data(None)
```

Comment:
```suggestion
        self.hass.loop.call_soon_threadsafe(self.coordinator.async_set_updated_data, None)
```

Let's discard the rest of change and update this line. Thanks for your contribution.